src/shared/fstab-util.c

```diff
@@ -105,6 +105,33 @@ static int fstab_is_same_node(const char *what_fstab, const char *path) {
         return false;
 }
 
+int fstab_has_mount_point_prefix_strv(char * const *prefixes) {
+        _cleanup_endmntent_ FILE *f = NULL;
+
+        assert(prefixes);
```

Comment:
drop this. we generally treat NULL strv equivalent to the empty strv

Reply:
I think it's better to leave it there. If this is called, and since this our only argument, we should assume that caller did want something from us and if it's empty I'd say it's a programming error...